apiserver/cmd/apiserver/server/watch.go

```diff
@@ -81,6 +84,19 @@ func WatchExtensionAuth(ctx context.Context) (bool, error) {
 					n := new.(*corev1.ConfigMap)
 					// Only detect as changed if the version has changed
 					if o.ResourceVersion != n.ResourceVersion {
+						changedKey := findFirstDifferingKey(o, n)
+						var oldVal, newVal string
+						if changedKey != "" {
+							oldVal = o.Data[changedKey]
+							newVal = o.Data[changedKey]
+						}
+						logrus.WithFields(logrus.Fields{
+							"oldResourceVersion": o.ResourceVersion,
+							"newResourceVersion": n.ResourceVersion,
+							"changedDataKey":     changedKey,
+							"oldKeyValue":        oldVal,
+							"newKeyValue":        newVal,
```

Comment:
We mustn't log sensitive values, and I think these come from a secret which by default makes them sensitive (unless carefully reasoned otherwise).

Reply:
Sorted now, thanks